services/account/k6-test/README.md

```diff
@@ -1,7 +1,7 @@
 # Generated k6 script
 
-The `account-service-load.js` file contains most of the Swagger/OpenAPI specification and you can customize it to your needs.
-The `health-check.js` file contains a simple health check script that can be used to check the health of the service.
+The `account-service-load.k6.js` file contains most of the Swagger/OpenAPI specification and you can customize it to your needs.
```

Comment:
First place this shows up...

So why?

- I needed to be able to know when something was specifically for k6
- This felt easier than trying to put all non-k6 test files into a subdirectory